Remove the relative path for libnettle.so so the test
program can find it.
Relative paths are not suitable, as the folder strucure for ptest
is different from the one expected by the nettle testsuite.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Juro Bystricky <juro.bystricky@intel.com>
Signed-off-by: Mingli Yu <mingli.yu@windriver.com>

--- a/testsuite/dlopen-test.c	2016-10-01 00:28:38.000000000 -0700
+++ b/testsuite/dlopen-test.c	2017-10-13 11:08:57.227572860 -0700
@@ -9,7 +9,7 @@
 main (int argc UNUSED, char **argv UNUSED)
 {
 #if HAVE_LIBDL
-  void *handle = dlopen ("../libnettle.so", RTLD_NOW);
+  void *handle = dlopen ("libnettle.so", RTLD_NOW);
   int (*get_version)(void);
   if (!handle)
     {
